src/main/resources/assets/twilightforest/lang/fr_fr.json

```diff
@@ -402,92 +527,101 @@
+  "block.twilightforest.sort_button": "Bouton en trieur",
+  "block.twilightforest.sort_fence": "Barrière en trieur",
+  "block.twilightforest.sort_gate": "Portillon en trieur",
+  "block.twilightforest.sort_plate": "Plaque de pression en trieur",
+  "block.twilightforest.sort_trapdoor": "Trappe en trieur",
+  "block.twilightforest.sort_door": "Porte en trieur",
+  "block.twilightforest.sort_sign": "Panneau en trieur",
+  "block.twilightforest.sort_wall_sign": "Panneau mural en trieur",
+
+  "block.steeleaf.tooltip": "Réduit les dégâts de chute de 25%",
+  "block.arctic.tooltip": "Réduit les dégâts de chute de 90%",
+  "block.knightmetal.tooltip": "Fonctionne comme un cactus plus fort",
+
+  "block.twilightforest.root_strand": "Brins de racine",
+  "block.twilightforest.fallen_leaves": "Tas de feuilles",
+  "block.twilightforest.liveroot_block": "Racines vivantes",
+  "block.twilightforest.trophy": "Trophée de la Forêt Twilight",
+  "block.twilightforest.trophy_pedestal": "Piédestal à trophée",
+  "block.twilightforest.hydra_trophy": "Trophée de l'Hydre",
+  "block.twilightforest.naga_trophy": "Trophée du Naga",
+  "block.twilightforest.lich_trophy": "Trophée de la Lich",
+  "block.twilightforest.ur_ghast_trophy": "Trophée de l'Ur-Ghast",
+  "block.twilightforest.yeti_trophy": "Trophée du Yéti Alpha",
+  "block.twilightforest.snow_queen_trophy": "Trophée de la Reine des Neiges",
+  "block.twilightforest.canopy_bookshelf": "Bibliothèques de canopée",
+
+  // "Structures",
   "structure.twilight_portal": "Portail de la Twilight",
-  "structure.hedge_maze": "Labyrinthe",
+  "structure.hedge_maze": "Labyrinthe de haies",
   "structure.hollow_hill": "Colline creuse",
   "structure.quest_grove": "Jardin de quête",
   "structure.mushroom_tower": "Château de champignons",
-  "structure.naga_courtyard": "Cour du Nâga",
-  "structure.lich_tower": "Tour du Liche",
+  "structure.naga_courtyard": "Cour du Naga",
+  "structure.lich_tower": "Tour du Lich",
   "structure.minotaur_labyrinth": "Labyrinthe du Minotaure",
   "structure.hydra_lair": "Repaire de l'Hydre",
-  "structure.goblin_stronghold": "Forteresse du globin",
+  "structure.goblin_stronghold": "Forteresse du gobelin",
   "structure.dark_tower": "Tour noire",
   "structure.yeti_cave": "Cave du Yéti",
-  "structure.aurora_palace": "Palace d'aurore",
+  "structure.aurora_palace": "Palace de l'Aurore",
   "structure.troll_cave_and_cloud_castle": "Cave du Troll et Maison dans les Nuages",
   "structure.final_castle": "Château final",
 
-  "_comment": "Items",
-  "item.twilightforest.naga_scale": "Écailles de Nâga",
-  "item.twilightforest.naga_chestplate": "Plastron en écailles de Nâga",
-  "item.twilightforest.naga_leggings": "Jambières en écailles de Nâga",
+  // "Items",
+  // "item.twilightforest.boarkchop": "Côtelette de Boark",
+  // "item.twilightforest.magic_map_focus": "?? de carte magique",
+  // "item.twilightforest.maze_map_focus": "?? de carte de labyrinthe",
```

Comment:
can I ask why these 3 are commented out? Do you need something clarified with them?

Reply:
well, boarkchop is supposed to be a play on words or something, @Drullkus made it, so he may need to clarify. Im not entirely sure how that would be translated. As for the focus ones, you may have to ask someone else for those as well. Im not quite sure how to describe them best, but they act like compasses in the normal map recipe. I like to think of them as the part that guides you on the map. Again, I may not be the best person to ask about it

Reply:
Boarkchop follows from a terrible pun, where Porkchop renamed as "Boarkchop" may randomly drop from Boars instead of regular Porkchop. ~~I appreciate the credit for the pun but that's actually Benimatic~~ lol